src/cmd/linuxkit/run_qemu.go

```diff
@@ -54,6 +55,19 @@ const (
 	qemuNetworkingDefault        = qemuNetworkingUser
 )
 
+var (
+	defaultArch string
+)
+
+func init() {
+	switch runtime.GOARCH {
+	case "arm64":
+		defaultArch = "aarch64"
+	default:
+		defaultArch = "x86_64"
```

Comment:
maybe explicit case x86 here and a "huh" in the default case? Or maybe leave as `""` so that if you run linuxkiut on e.g. powerpc you have to pick if you want to emulate arm or x86 (until there is a ppc port!)